.prettierignore

```diff
@@ -0,0 +1,4 @@
+# ignore all files
+*
```

Comment:
not sure we need this file. there are some files outside of src which are fine to pretty. let's make this file blank

Reply:
👍